internal/core/algorithm/ngt/Makefile

```
@@ -15,11 +15,21 @@
 
 bench:
+	CC="$(CC)" \
+	CXX="$(CXX)" \
+	AR="$(AR)" \
+	NM="$(NM)" \
+	RANLIB="$(RANLIB)" \
 	GOARCH=$(GOARCH) \
 	GOOS=$(GOOS) \
 	go test -count=5 -run=NONE -bench . -benchmem
 
 bench-lite:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)